ASPNetCoreMasters/Controllers/ItemsController.cs

```diff
@@ -10,32 +10,57 @@
 
 namespace ASPNetCoreMasters.Controllers
 {
+    [ApiController]
+    [Route("[controller]")]
     public class ItemsController : ControllerBase
     {   
-        public int Get(int id)
+        [HttpGet]
+        public IActionResult GetAll()
         {
-            //var item = new ItemService();
-            //var result = item.GetAll(id);
+            return Ok("GetAll route");
+        }
 
-            return id;
+        [HttpGet("{itemId}")]
+        public IActionResult Get(int itemId)
+        {
+            var itemService = new ItemService();
+            itemService.GetItem(itemId);
+            return Ok($"Get {itemId}");
+        }
+
+        [HttpGet("GetByFilters")]
```

Comment:
currently this action will work with the uri "/items/GetByFilters" but we want "/items/filterBy" as per requirements. Minor change needed